spec/card_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'rails_helper'
+require 'support/factory_girl.rb'
```

Comment:
@davetoxa ok, is it because it's automatically part of config? was following the documentation